src/misc/support.cpp

```diff
@@ -392,3 +392,25 @@ const std_fs::path &GetExecutablePath()
 	}
 	return exe_path;
 }
+
+const Bit8u DOS_DATE_months[] = {0,  31, 28, 31, 30, 31, 30,
+                                 31, 31, 30, 31, 30, 31};
+
+bool is_date_valid(uint32_t year, uint32_t month, uint32_t day)
```

Comment:
I think we can do all three args `const` here too :+1: 

(In function bodies, try to set arguments `const` to show that the argument is read-only.  Otherwise, the reason needs to be on the lookout if we also use those args are variables and change them.)